README.md

```diff
@@ -276,7 +276,7 @@ or
 IoTDB> exit
 ```
 
-For more on what commands are supported by IoTDB SQL, see [Chapter 5: IoTDB SQL Documentation](https://iotdb.apache.org/#/Documents/0.8.0/chap5/sec1).
+For more on what commands are supported by IoTDB SQL, see [Chapter 5: IoTDB SQL Documentation](https://iotdb.apache.org/#/Documents/0.9.0/chap5/sec1).
 
 ### Stop IoTDB
```

Comment:
The title of this chapter has changed. The correct title and link should be `[SQL Reference](https://iotdb.apache.org/#/Documents/progress/chap5/sec4)`

Reply:
fixed to https://iotdb.apache.org/#/Documents/0.9.0/chap5/sec4